tests/data/examples/v06/image_example.json

```diff
@@ -4,20 +4,38 @@
     "multiscales": [
       {
         "name": "example",
-        "axes": [
-          { "name": "t", "type": "time", "unit": "millisecond" },
-          { "name": "c", "type": "channel" },
-          { "name": "z", "type": "space", "unit": "micrometer" },
-          { "name": "y", "type": "space", "unit": "micrometer" },
-          { "name": "x", "type": "space", "unit": "micrometer" }
+        "type": "gaussian",
+        "coordinateSystems": [
+          {
+            "name": "example",
```

Comment:
Could we rename this "coord_sys_1" or similar so it's a bit less generic than just "example" (and the same for other coordinate systems in the metadata files)

Reply:
Good point.